blueocean-dashboard/src/main/js/components/RunDetails.jsx

```diff
@@ -72,40 +73,28 @@ class RunDetails extends Component {
             return null;
         }
 
-        const {
-            router,
-            location,
-            params: {
-                organization,
-                branch,
-                runId,
-                pipeline: name,
-            },
-        } = this.context;
-
-        const baseUrl = buildRunDetailsUrl(organization, name, branch, runId);
-
-        /* eslint-disable arrow-body-style */
-        const currentRun = this.props.runs.filter((run) => {
-            return run.id === runId &&
-                decodeURIComponent(run.pipeline) === branch;
-        })[0];
+        const { router, location, params } = this.context;
```

Comment:
Thank you for doing this ... that abusive use of destructing was something that irritated me intensely !!